Description: Fix spelling mistakes found by lintian
Author: Peter Colberg <peter@colberg.org>
Forwarded: https://github.com/hlandau/acme/pull/134
Applied-Upstream: https://github.com/hlandau/acme/commit/d34f9600b21e7dec7f7053d1033c33386dea8848
Last-Update: 2016-03-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cmd/acmetool/doc.go
+++ b/cmd/acmetool/doc.go
@@ -60,7 +60,7 @@ certificates from ACME server such as Le
 idempotency and the minimisation of state.
 
 You use acmetool by configuring targets (typically using the "want") command.
-acmetool then requests certificates as necessary to satisify the configured
+acmetool then requests certificates as necessary to satisfy the configured
 targets. New certificates are requested where existing ones are soon to expire.
 
 acmetool stores its state in a state directory. It can be specified on
--- a/cmd/acmetool/main.go
+++ b/cmd/acmetool/main.go
@@ -56,7 +56,7 @@ var (
 
 	wantCmd       = kingpin.Command("want", "Add a target with one or more hostnames")
 	wantReconcile = wantCmd.Flag("reconcile", "Specify --no-reconcile to skip reconcile after adding target").Default("1").Bool()
-	wantArg       = wantCmd.Arg("hostname", "hostnames for which a certicate should be obtained").Required().Strings()
+	wantArg       = wantCmd.Arg("hostname", "hostnames for which a certificate should be obtained").Required().Strings()
 
 	unwantCmd = kingpin.Command("unwant", "Modify targets to remove any mentions of the given hostnames")
 	unwantArg = unwantCmd.Arg("hostname", "hostnames which should be removed from all target files").Required().Strings()
